src/sync/streaming/parseUtils.ts

```diff
@@ -75,3 +76,22 @@ export function isInBitmap(bitmap: Uint8Array, hash64hex: string) {
   const offset = index % 8;
   return (bitmap[internal] & 1 << offset) > 0;
 }
+
+/**
+ * Parse feature flags notifications for instant feature flag updates
+ *
+ * @param {ISplitUpdateData} data
+ * @returns {KeyList}
+ */
+export function parseFFUpdatePayload(data: ISplitUpdateData): KeyList | undefined {
+  const avoidPrecisionLoss = false;
+  try {
```

Comment:
I would not handle exceptions here, because the caller need to know if the decompression fail or not. I would do like other utility functions, like  `parseBitmap` and `parseKeyList`.